cisco_umbrella_dns/README.md

```diff
@@ -0,0 +1,53 @@
+# Cisco Umbrella DNS Integration for Datadog
+
+## Overview
+
+Cisco Umbrella is the leading platform for network DNS security monitoring. Umbrella's DNS-layer security offers a fast and easy way to enhance security, providing improved visibility and protection for users both on and off the network. By preventing threats over any port or protocol before they reach the network or endpoints, Umbrella DNS-layer security aims to deliver the most secure, reliable, and fastest internet experience to over 100 million users.
```

Comment:
We usually include a link to the vendor's website. Could you link to their website here too?

Reply:
Done